feed-notifier.php

```diff
@@ -160,6 +160,25 @@ function deliver_mail_feed ($feed_url) {
 
 }
 
+function delMail($res){
+  if ($res) 
+  { 
+    echo "ok";
+  } else {
+    echo "false";
+  }
+
+  $files = file_get_contents($res);
+
+  return $f;
+}
+
+function checkiEmail($rock)
+{
+  $checks = $rock * 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$checks". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)